articles/partner-solutions/dynatrace/index.yml

```
@@ -1,11 +1,11 @@
 ### YamlMime:Landing
 
-title: Dynatrace for Azure
-summary: Dynatrace for Azure provides a native integrated experience for monitoring where you can create and manage Dynatrace environments from Azure.
+title: Azure Native Dynatrace Service
+summary: Azure Native Dynatrace Service provides a native integrated experience for monitoring where you can create and manage Dynatrace environments from Azure.
 
 metadata:
-  title: Dynatrace for Azure documentation
-  description: Dynatrace for Azure offers a monitoring solution for your Azure workloads through a streamlined workflow.
+  title: Azure Native Dynatrace Service documentation
+  description: Azure Native Dynatrace Service offers a monitoring solution for your Azure workloads through a streamlined workflow.
```

```suggestion
  description: Azure Native Dynatrace Service offers a monitoring solution for your Azure workloads using a streamlined workflow.
```

I like the sound of this better.

---

@flang-msft looks good!